build-farm/make-adopt-build-farm.sh

```diff
@@ -128,9 +128,16 @@ then
   echo "Detecting boot jdk for: ${JAVA_TO_BUILD}"
   echo "Found build version: ${JAVA_FEATURE_VERSION}"
   JDK_BOOT_VERSION=$(( JAVA_FEATURE_VERSION - 1 ))
+  if [ "${JAVA_FEATURE_VERSION}" == "11" ] && [ "${VARIANT}" == "openj9" ]; then
+    # OpenJ9 only supports building jdk-11 with jdk-11
+    JDK_BOOT_VERSION="11"
+  fi
 fi
 echo "Required boot JDK version: ${JDK_BOOT_VERSION}"
 
+export REQUIRED_BOOT_JDK_VERSION
+REQUIRED_BOOT_JDK_VERSION="${JDK_BOOT_VERSION}"
```

Comment:
What's the reason for creating a new variable instead of just using `JDK_BOOT_VERSION`?

Reply:
because it was being export'd and I wanted the name to be very clear as in "REQUIRED". "JDK_BOOT_VERSION" is rather anonymously named! IMO

Reply:
Hmm not sure "because it was being exported" is a good reason to create a new variable (Does it even need to be exported?) which adds complexity. We should perhaps consider changing the name of the existing one.

Reply:
ok no worries, i'll change

Reply:
Not sure if we need to change the existing one or add a new one. Yes, for some JDK we need different JDK_BOOT_VERSION from other but IMO JDK_BOOT_VERSION  still works. It is quite clear and straightforward IMO.

Reply:
done